lib/utils/constant_icons.dart

```diff
@@ -1,54 +1,52 @@
 import 'package:flutter/widgets.dart';
 
 class AppIcon {
-  static final int fabTweet = 0xf029;
-  static final int messageEmpty = 0xf187;
-  static final int messageFill = 0xf554;
-  static final int search = 0xf058;
-  static final int searchFill = 0xf558;
-  static final int notification = 0xf055;
-  static final int notificationFill = 0xf019;
-  static final int messageFab = 0xf053;
-  static final int home = 0xf053;
-  static final int homeFill = 0xF553;
-  static final int heartEmpty = 0xf148;
-  static final int heartFill = 0xf015;
-  static final int settings = 0xf059;
-  static final int adTheRate = 0xf064;
-  static final int reply = 0xf151;
-  static final int retweet = 0xf152;
-  static final int image = 0xf109;
-  static final int camera = 0xf110;
-  static final int arrowDown = 0xf196;
-  static final int blueTick = 0xf099;
+  static const _fontFamily = 'TwitterIcon';
 
-  static final int link = 0xf098;
-  static final int unFollow = 0xf097;
-  static final int mute = 0xf101;
-  static final int viewHidden = 0xf156;
-  static final int block = 0xe609;
-  static final int report = 0xf038;
-  static final int pin = 0xf088;
-  static final int delete = 0xf154;
+  static final fabTweet = IconData(0xf029, fontFamily: _fontFamily);
```

Comment:
I thought instead of writing the whole IconData(......) thing, whenever dev want's to use the CustomIcon they will write the codes itself.
**Anyways thanks for doing**